### ISSUE TITLE
HW6 problem 3

### ISSUE THREAD

**Issue by bltnynk** [swtv-kaist/cs458-fall22#18]
Do we need to include `./tcas-mod` as the prefix in our generated test cases .txt file?
<img width="923" alt="image" src="https://user-images.githubusercontent.com/61192617/205493060-aea43714-d895-4f87-863b-e8926a153336.png">

**Comment by moonzoo** [comment]
Yes, please.  TA can run all your test inputs conveniently by doing that.
